### PR TITLE
update add flashcard  to group command

### LINKED ISSUE #158: [PE-D] add flashcard to group command input
As the list-group command takes in both group name and group index, it would be more consistent, and probably more convenient if the add command accepts both group name and group index too, instead of just group name.

-------------
Labels: `severity.VeryLow` `type.FeatureFlaw`
original: JeremiasLiew/ped#8

### PR DESCRIPTION
**Pull request by sliu107**
… index

### CODE REVIEW

File: src/test/java/seedu/tp/commands/AddFlashcardToGroupCommandTest.java [hunk 1]
```diff
@@ -121,10 +123,25 @@ public void addFlashcardToGroupCommand_unrecognizedFlashcardGroupType_throwsExce
         GroupCommand groupCommand = new GroupCommand(groupFactory, originalGroupList);
         groupCommand.execute();
 
-        AddFlashcardToGroupCommand addFlashcardToGroupCommand = getAddFlashcardToGroupCommandWithInput(
+        //Test case1: Given an unionized group name
```
Comment (g-lilian) [inline src/test/java/seedu/tp/commands/AddFlashcardToGroupCommandTest.java]: What does "unionized" mean? O.o

**Review by g-lilian — APPROVED** [review]
Looks good, perhaps you can update text-ui-test as well? By adding in cases where index is used instead of group name.

File: src/test/java/seedu/tp/commands/AddFlashcardToGroupCommandTest.java (hunk 1 above)
Comment (Zhilin-Huang) [reply src/test/java/seedu/tp/commands/AddFlashcardToGroupCommandTest.java]: Same question haha

**Review by Zhilin-Huang — CHANGES_REQUESTED** [review]
Remember to also update UG for this change

**Pull request merged**